console.js

```diff
@@ -0,0 +1,37 @@
+const repl = require('repl'),
+  fs = require('fs'),
+  pjson = require('./package.json'),
+  orm = require('./app/orm'),
+  testFolder = './tests/';
+
+const convertFunctionToAsync = f => {
+  return async (...args) => {
+    const result = await f(...args);
+    console.log(result);
```

Comment:
If you want to have some nice you can print as the following:
```js
console.log(JSON.stringify(result, null, 2));
```

Reply:
I don't see the need of printing this.

Reply:
I couldn't find away to print it in the console. Returning the result does not work. Don't know if it can be done with any other repl method.